test/common.py

```diff
@@ -242,7 +242,12 @@ def flaky(note=''):
 
   def decorated(f):
     @wraps(f)
-    def modified(*args, **kwargs):
+    def modified(self, *args, **kwargs):
+      # Browser tests have there own method of retrying tests.
+      if self.is_browser_test():
+        self.extra_tries = EMTEST_RETRY_FLAKY
+        return f(self, *args, **kwargs)
```

Comment:
Since this is basically a seperate decorator perhaps we should just define it in `test_browser.py`.

We could give it a unique name, or keep the the same, and it could take an integer `extra_tries`?